### ISSUE TITLE
ImportError: cannot import name 

### ISSUE THREAD

**Issue by Chokeey** [PatrickAlphaC/nft-mix#38]
Whenever I try to run my script with this command:
 'brownie run .\scripts\advanced_collectible\deploy_advanced.py',
I always get an error, I've tried multiple things to fix it and nothing seems to work.
**ERROR:**
ImportError: cannot import name 'AdvancedCollectible' from 'brownie' (C:\Python\Python39\lib\site-packages\brownie\__init__.py)

I've got the latest version of brownie and python.
![image](https://user-images.githubusercontent.com/90982478/133898362-0c25cb11-d59d-4b64-8886-defab08e39de.png)

Recently whilst trying to fix it another problem popped up, not sure if it's going anything to do with it.
![image](https://user-images.githubusercontent.com/90982478/133898398-82dbc066-db6c-44bd-9411-b54bc2d2efd1.png)

**Comment by PatrickAlphaC** [comment]
Your slashes are backwards:

```
brownie run scripts\advanced_collectible\deploy_advanced.py --network rinkeby
```
Change to 
```
brownie run scripts/advanced_collectible/deploy_advanced.py --network rinkeby
```

**Comment by Chokeey** [comment]
I've tried what you suggested, I'm still having the same exact error.

![image](https://user-images.githubusercontent.com/90982478/134061735-39e3378b-981b-47f0-85f0-8346b7cc1fef.png)

Is there any reason why "brownie" has a brown line underneath it?

**Comment by PatrickAlphaC** [comment]
What's the name of your contract in that file? It's not the name of the file that counts, it's the name of the contract. 

You can ignore the yellow swiggle.

**Comment by Chokeey** [comment]
![image](https://user-images.githubusercontent.com/90982478/134513878-80212a9f-fba0-4e51-992c-8d835ad1b337.png)
![image](https://user-images.githubusercontent.com/90982478/134513981-23363e9a-c5be-4da4-b4ba-637cc0941520.png)

**Comment by PatrickAlphaC** [comment]
No, the name of the contract. Inside your advanced_collectible.sol

**Comment by dev-bhaskar8** [comment]
![image](https://user-images.githubusercontent.com/43923435/136115584-da4181af-ad9a-475f-b8f9-267aea764751.png)
